ci/docker/runtime_functions.sh

```diff
@@ -1125,6 +1125,30 @@ nightly_tutorial_test_ubuntu_python2_gpu() {
     nosetests-3.4 --with-xunit --xunit-file nosetests_tutorials.xml test_tutorials.py --nologcapture
 }
 
+nightly_example_test_ubuntu_python3_gpu() {
```

Comment:
Please review everything in here. It seems like this has just been copy and pasted